### DISCUSSION TITLE
How to remove a PWM Channel?

### DISCUSSION THREAD

**Discussion by ediiiz** [arendst/Tasmota#16307] [Q&A] [answered]
Hello everyone,

I flashed my RGBW (2700K) bulbs with tasmota a couple of days ago. 
Everything works fine, but after configuring the Template i noticed that the bulb exposes 5 Channels but in reality it only has 4 Channels. (CCT Control not useable)
I searched everywhere, but I couldnt find an answer. 

Can i somehow remove the empty Channel?

This is my Template:
{"NAME":"SH 360","GPIO":[0,0,0,417,2912,416,0,0,0,2976,2944,0,0,1],"FLAG":0,"BASE":18}

https://templates.blakadder.com/swisstone_SH360.html

**Comment by barbudor** [comment, marked as answer]
In the configuration => Template page, remove the PWM 5

**Reply by ediiiz** [reply]
Hmm i dont have pwm 5.

I got:

GPIO03 | PWM 1 (Exposes 2 Channels, but the bulb only has warmwhite leds. If I remove this, both channels are gone)
GPIO04 | SM16716 CLK (RGB, if i remove RGB is gone)
GPIO13 | SM16716 PWR (RGB, if i remove RGB is gone)
GPIO14 | SM16716 DAT (RGB, if i remove RGB is gone)

**Reply by s-hadinger** [reply]
It looks like you have `PWM2` on GPIO3. Please remove it.

**Reply by ediiiz** [reply]
> It looks like you have `PWM2` on GPIO3. Please remove it.

Oh okay...i think i got it. 

Tasmota doesnt show the white channel without RGB White Split, didnt understand that correctly.
But HomeAssistant displays the White Channel. Thanks for the help and sorry for the dumb question.
